v2/third_party/uw-labs/lichen/METADATA

```diff
@@ -0,0 +1,16 @@
+name: "uw-labs/lichen"
+description:
+    "uw-labs/lichen is a license tool to analyze go binaries"
+
+third_party {
```

Comment:
Yes, I am using the internal package, but I didn't make any changes.

The file itself follows google third party metadata syntax and it's manually maintained. We don't necessarily need to be so strict with the format. I think it does cause some confusion.

I will just change its format to markdown to simplify.

Reply:
Re forking the library - I'm hesitant to fork since it makes the dependency upgrade process much more manual. Looking at the lichen code in more detail, IIUC it looks like this is effectively wrapping `go version -m`. If that's all we need, it might be simpler to just have our own func that accomplishes a similar result (I think we could use the code you already wrote here actually).

Re: third_party format - we should avoid using this the schema isn't widely accessible (it would be difficult for non-googlers to know what is expected here).